File: C/Daily Flash/Week1/Day6_Jan18/code_C/prog3.c
/*
 * Program 3 : Write a Program to print First 50 Odd Numbers .
Output: 1 3 5 . . . 99
*
*/

#include <stdio.h>

void main() {

	printf("First 50 odd Numbers:\n");
	int num =1;

	for(int i=1; i<= 50; i++){
		printf("%d ",num);
		num +=2;
	}
	printf("\n");
}

File: PYTHON/Daily Flash/Week2/Day1_Jan20/code/prog3.py
'''
Program 3 : Write a Program that accepts a n inte ger from user and prints
whether that number is odd or even .
Input: 2
Output: 2 is an Even Number
'''

num = int(input("Enter number: "))

if num%2 == 0:
    print(str(num)+" is an Even number")
else:
    print(str(num)+" is an Odd number")
